### PR TITLE
balances self surgery + fixes farmbots

### PR DESCRIPTION
**Pull request by cdb-is-not-good**
Self surgery is now 12.5% harder for organics and 50% harder for robots.

Self surgery is now 50% easier for Cht'Mants

Fixes; Farmbots putting ammonia into hydroponics trays along with water, thus creating space cleaner. Fertilizing now uses ez nutrient.

## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: code/modules/surgery/surgery.dm [hunk 1]
```diff
@@ -149,13 +149,18 @@
...
 		//For if a user is doing 'surgery' on their own prosthetic bodypart
 		if(nature == MODIFICATION_SILICON)
-			difficulty_adjust = 60
-			time_adjust = 20
+			difficulty_adjust = 90 //this is VERY complicated work to do with perfect sightlines and ergonomics - let alone without these.
+			time_adjust = 40
+
+
+	if(user.stats.getPerk(PERK_SCUTTLEBUG || PERK_ICHOR || PERK_CHITINARMOR))
+		difficulty_adjust = -80 //We feel no pain, and are pretty used to working on ourselves due to metal paranoia. Still slightly worse than letting someone else do, due to limited ability to see inside
```
Comment (Trilbyspaceclone) [inline code/modules/surgery/surgery.dm:162]:
```suggestion
		difficulty_adjust += -80 //We feel no pain, and are pretty used to working on ourselves due to metal paranoia. Still slightly worse than letting someone else do, due to limited ability to see inside
```

File: code/modules/surgery/surgery.dm [hunk 2]
```diff
@@ -149,13 +149,18 @@
...
+		time_adjust = -30
```
Comment (Trilbyspaceclone) [inline code/modules/surgery/surgery.dm:163]:
```suggestion
		time_adjust += -30
```

**Comment by benj8560** [comment]
I disapprove of this tbh. Took me 20+ tries to insert a augment at 30ish mechanical but likely the primary thing this was trying to nerf. Still did not seem to hard opening/closing panels with good tools. Big issue tho? You can't get ''better'' cables and we are likely back to it being nearly impossible to self repair major burns. The odds are about the same for self inserting and internal cable repair from what I've seen in the past

**Comment by cdb-is-not-good** [comment]
> I disapprove of this tbh. Took me 20+ tries to insert a augment at 30ish mechanical but likely the primary thing this was trying to nerf. Still did not seem to hard opening/closing panels with good tools. Big issue tho? You can't get ''better'' cables and we are likely back to it being nearly impossible to self repair major burns. The odds are about the same for self inserting and internal cable repair from what I've seen in the past. I'll honestly agree with self auging being very busted and proly has the nerf coming but we need cable repair right where it was before difficult but not an absolute nightmare.

Perhaps it was my luck but this by no means made it 'impossible' to repair big burns, you still get functionally unlimited tries with no real downside to just spamming it.

**Comment by benj8560** [comment]
''still get functionally unlimited tries'' so....then why even do this? Just leave it as it was. All you are doing is forcing players to sit there and spam click a button rather then play the game. The LAST change to synth Self surgery was literally making it easier. We have done this flip a few times now why are we doing circles? I don't see this changing anything aside from forcing players to spam chat messages and again not play the game.

**Comment by cdb-is-not-good** [comment]
Ideally the balance here should be that it's possible, but not preferable to do your own repairs/surgery.

**Pull request merged**